ui/jobs/combo_tracker.ts

```diff
@@ -1,7 +1,19 @@
 import { kAbility, kComboBreakers } from './constants';
 
+type StartMap = Record<string, {
+  id?: string;
+  next?: StartMap;
```

Comment:
I think these are both required.  Am I misunderstanding? `next` might be `{}` but that's a valid `StartMap`.

Reply:
Yes, you're right =_=